playwright/README.md

```diff
@@ -0,0 +1,20 @@
+## Troubleshooting Failed Integration Tests in CI
+
+### Single Test Failure
+
+There are stored artifacts in the GitHub CI. Go to the failed job and there is
+a link "Summary". At the bottom of the page there is "Artifacts" section which
+contains the `y2log` and also `trace.zip` file. The trace can be browsed using
+the playwright too locally or at page https://trace.playwright.dev/ to get
```

Comment:
tool?

Reply:
probably. @lslezak you add it. So if you can confirm it